build/golint.sh

```diff
@@ -22,7 +22,7 @@ TIMEOUT="10m"
 
 echo "Running golangci-lint..."
 
-golangci-lint run --timeout ${TIMEOUT} --skip-dirs ${SKIP_DIR_REGEX} -E maligned,whitespace,gocognit,unparam -e '`ctx` is unused'
+golangci-lint run --timeout ${TIMEOUT} --skip-dirs ${SKIP_DIR_REGEX} -E govet,whitespace,gocognit,unparam -e '`ctx` is unused'
```

Comment:
Why is this change required?

Reply:
maligned is deprecated in current version of golangci-lint we use.

Reply:
Is `govet` a replacement of this?

Reply:
It's good to provide references or links for such changes
```
maligned ⚙️ ⚠ | The repository of the linter has been archived by the owner. Replaced by govet 'fieldalignment'.
```
https://github.com/mdempsky/maligned

Reply:
added comment